qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -406,16 +406,19 @@ def _default_transpile_options(cls) -> Options:
         return options
 
     def enable_restless(
-        self, rep_delay: Optional[float] = None, override_processor_by_restless: bool = True
+        self,
+        rep_delay: Optional[float] = None,
+        override_processor_by_restless: bool = True,
+        ignore_t1_check: bool = False,
```

Comment:
This is not the scope of this PR but this method overrides api docs. Did you check if the arguments are documented properly?

Reply:
I added the args to be on the safe side here: [716b82c](https://github.com/Qiskit/qiskit-experiments/pull/820/commits/716b82c7adbe88e38e7a08f7d021988a0619630b)